### PR TITLE
entangled states work on the notebook

### PR DESCRIPTION
**Pull request by joshcombes**

### CODE REVIEW

**Comment by kylegulshen** [comment]
I can't quite figure out why this is failing. It doesn't seem to have to do with the std_errs--I think the Nan is actually appearing in the expectations, but I don't see how that could happen.

**Comment by joshcombes** [comment]
will partially close https://github.com/rigetti/forest-benchmarking/issues/98

File: examples/entangled_states.ipynb [hunk 1]
```diff
@@ -4,21 +4,28 @@
...
    "source": [
-    "# Entangled State Example\n",
+    "# Entangled States\n",
     "\n",
-    "This example shows how to enact and measure a graph state on a set of connected qubits that form a graph with the qubits as nodes and pairs of coupled qubits as edges. "
+    "In this notebook we explore the subset of methods in `entangled_states.py` that are related specifically to [graph states](https://en.wikipedia.org/wiki/Graph_state). Although it is worth noting that the module also alows one to create [GHZ](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state) states.\n",
+    "\n",
+    "\n",
+    "Graph states are a very simple entangled state construct as there are only two steps:\n",
+    "1. prepare all qubits on the desired lattice in the $|+\\rangle$ state \n",
```
Comment (kylegulshen) [inline examples/entangled_states.ipynb]: The \rangle is showing up quite large.

File: examples/entangled_states.ipynb [hunk 2]
```diff
@@ -4,21 +4,28 @@
...
+    "2. Do a CZ gate between all the edges on the lattice that have two qubit gates\n",
+    "\n",
+    "There is a very simple way to benchmark short depth circuits on a [NISQ](https://arxiv.org/abs/1801.00862) hardware. First choose a random connected subgraph on $N$ qubits computed the fidelity of the graph state to the ideal and repeat $K$ times. Do this for increasingly large graphs. Then plot the mean  fidelity as a function of the number of qubits."
```
Comment (kylegulshen) [inline examples/entangled_states.ipynb]:
```suggestion
    "There is a very simple way to benchmark short depth circuits on [NISQ](https://arxiv.org/abs/1801.00862) hardware. First choose a random connected subgraph on $N$ qubits, compute the fidelity of the graph state to the ideal and repeat $K$ times. Do this for increasingly large graphs. Then plot the mean fidelity as a function of the number of qubits."
```

File: examples/entangled_states.ipynb (hunk 1 above)
Comment (joshcombes) [reply examples/entangled_states.ipynb]: That is weird. It looks fine in my notebook. hmmm ill let it slide now.

**Pull request merged**